tests/socketio_win32_ut/CMakeLists.txt

```diff
@@ -1,24 +0,0 @@
-#Copyright (c) Microsoft. All rights reserved.
```

Comment:
Thanks, fixing. it.

---
In reply to: [105763370](https://github.com/Azure/azure-uamqp-c/pull/45#discussion_r105763370) [](ancestors = 105763370)